src/test/java/jp/co/soramitsu/sora/validation/KeyValidatorTest.java

```diff
@@ -12,7 +13,7 @@
   @ParameterizedTest
   @CsvFileSource(resources = {"/incorrectDids.csv", "/incorrectKeys.csv"}, numLinesToSkip = 1)
   public void testInvalid(String did) {
-    ddo.setId(did);
+    ddo.getPublicKey().get(0).setId(URI.create(did));
```

Comment:
can we implement method which will incapsulate getting 0-index element from list?

Reply:
Of course we can, but why do we need it?